README.md

```diff
@@ -498,13 +498,13 @@ The [Google Places Search API](https://developers.google.com/places/web-service/
 #### LocationIQ (`:location_iq`)
 
 * **API key**: required
-* **Quota**: 6 request/second (30k req/day), then [ability to purchase more](http://locationiq.org/#pricing)
+* **Quota**: 60 requests/minute (2 req/day, 10k req/day), then [ability to purchase more](http://locationiq.org/#pricing)
```

Comment:
I think there's a typo here (req/sec instead of req/day?)

Reply:
Corrected ;)